HelpSource/Classes/Matrix.schelp

```diff
@@ -241,6 +241,24 @@ ARGUMENT::
 colB
 
 
+METHOD:: zeroWithin
```

Comment:
In sclang, this method is called `thresh`. In principle, `Matrix` should support this method out of the box. This works fine:

```
a = Matrix.with(Array.rand(9, -1.0, 1.0).clump(3));
a.thresh(0.33);
```

It would be good to check all the methods in `Array` and `SequenceableColection` and see if they work as expected for `Matrix`.

Reply:
Actually... I've just noticed that the description for Help is not exactly correct.

`if (value <= within) { 0 } { value }`

It _should_ read:

`if (value.abs <= within) { 0 } { value }`

This returns a different result from `thresh`.

For instance:

```
a = Matrix.with([ 
	[ -4, 1, 6 ],
	[ 5, 6, -4 ],
	[ -6, -1, -9 ]
 ]);
a.thresh(3);
```

Returns:
```
Matrix[ 
	[ 0, 0, 6 ],
	[ 5, 6, 0 ],
	[ 0, 0, 0 ]
 ]

```

Whereas:
`a.zeroWithin(3)`

Returns:

```
Matrix[ 
	[ -4, 0, 6 ],
	[ 5, 6, -4 ],
	[ -6, 0, -9 ]
 ]

```

Reply:
Relatedly, due to the relationship with `thresh`, the method could be named `threshAbs`?

If that sounds like a good name, then @mtmccrea, we'd want to reflect this (change) in [MatrixArray](https://gitlab.com/dxarts/projects/matrixarray.quark), too.

Reminder to self: also check [SignalBox](https://gitlab.com/dxarts/projects/SignalBox.quark).